content/blog/2020-08-19-train-a-sequence-model-with-poutyne/index.en.md

```diff
@@ -0,0 +1,608 @@
+---
+title: Train a Recurrent Neural Network (RNN) with Poutyne
```

Comment:
*ing in titles: http://www.davidnicholson.it/language/on-ing-forms-in-titles.html 
```suggestion
title: Training a Recurrent Neural Network (RNN) with Poutyne
```